packages/fast-development-site-react/src/components/site/configuration-panel.tsx

```diff
@@ -0,0 +1,201 @@
+import * as React from "react";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import { IDevSiteDesignSystem } from "../design-system";
+import Form from "@microsoft/fast-form-generator-react";
+import manageJss, { ComponentStyles, IManagedClasses } from "@microsoft/fast-jss-manager-react";
+
+export enum TabType {
+    presets = "Presets"
+}
+
+export interface IConfigurationPanelProps {
+    formChildOptions: any;
+    schema: any;
+    data: any;
+    onChange: any;
+    activeTab?: TabType;
+}
+
+export interface IConfigurationPanelState {
+    activeTab: TabType;
+}
+
+export interface IConfigurationPanelManagedClasses {
+    configurationPanel: string;
+    configurationPanel_controls: string;
+    configurationPanel_controls_tabs: string;
```

Comment:
I think this should just be `configurationPanel_tabs` or `configurationPanel_controlsTabs`